src/SDK.ts

```diff
@@ -313,6 +318,12 @@ export default class SDK implements ISDK {
         const { data, headers } = response;
         this.setAuthCodeNonce(headers);
 
+        if (headers) {
+          if (headers[OmnichannelHTTPHeaders.ocSessionId.toLowerCase()]) {
```

Comment:
is there a risk this could be null or undefined/. headers[OmnichannelHTTPHeaders.ocSessionId

Reply:
It has a null/undefined check on whether the `headers` exists, then whether `Oc-Sessionid` header exists, then assign it accordingly.